### DISCUSSION TITLE
Deprecated `use` still shows up in `useInstantSearch` docs

### DISCUSSION THREAD

**Discussion by avremel** [algolia/instantsearch#5815] [General] [answered]
The `useInstantSearch` [docs](https://www.algolia.com/doc/api-reference/widgets/use-instantsearch/react/#widget-param-use) has this example:

```jsx
import { myMiddleware } from './myMiddleware';

function MyMiddleware() {
  const { use } = useInstantSearch();

  useEffect(() => {
    return use(myMiddleware);
  }, [use]);

  return null;
}
```
`use` no longer works in version 7 of `react-instantsearch`.

Docs for `insights` has the correct replacement: `addMiddleWares` - https://www.algolia.com/doc/api-reference/widgets/insights/react/.

**Comment by Haroenv** [comment, marked as answer]
Thanks, we must have missed this in the migration. Making a PR to replace it now
